.github/workflows/docker-push.yaml

```diff
@@ -0,0 +1,32 @@
+name: image-publisher
+
+on:
+  push:
+    branches: [ '*' ]
+
+jobs:
+  docker-push:
+    name: docker-build
+    runs-on: ubuntu-latest
+    if: github.repository == 'red-hat-storage/odf-operator'
+    strategy:
+      fail-fast: false
+      matrix:
+        go: ["1.15"]
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go }}
+
+    - uses: actions/checkout@v2
+
+    - name: Login to Quay
+      uses: docker/login-action@v1
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_ROBOT_TOKEN }}
+
+    - name: Build and push docker images
+      run: |
+        IMAGE_TAG=${GITHUB_REF#refs/heads/} make docker-build docker-push bundle-build bundle-push catalog-build catalog-push
```

Comment:
any particular reason for putting all the commands in one line instead of having multiple invocations of make?

Reply:
Not really it is just how I run it and like it.

Reply:
The only thing in an automation like this is:
The more separate these steps are, the easier it will be to see where stuff fails if it fails, i.e. separate steps for each make target might make troubleshooting easier.

Reply:
updated